xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/BittrexExchange.java

```diff
@@ -70,7 +78,16 @@ public void remoteInit() throws IOException {
       synchronized (INIT_LOCK) {
         if (bittrexSymbols.isEmpty()) {
           bittrexSymbols = ((BittrexMarketDataServiceRaw) marketDataService).getBittrexSymbols();
-          BittrexAdapters.adaptMetaData(bittrexSymbols, exchangeMetaData);
+          List<BittrexCurrency> bittrexCurrencies =
+                  ((BittrexMarketDataServiceRaw) marketDataService).getBittrexCurrencies();
+          Map<CurrencyPair, Fee> dynamicTradingFees = null;
+          try {
+            dynamicTradingFees = accountService.getDynamicTradingFees();
+          } catch (BittrexException | IOException e) {
+              EXCHANGE_LOGGER.log(Level.WARNING,
+                    "Error during remote init, can not fetch trading fees. May be missing auth tokens ?");
```

Comment:
pass the exception to the log:

`log.warn("...", e)`

Reply:
Made the regarding changes in the latest commit.